cfy_manager/constants.py

```diff
@@ -31,7 +31,10 @@
 SUDOERS_INCLUDE_DIR = '/etc/sudoers.d'
 CLOUDIFY_SUDOERS_FILE = join(SUDOERS_INCLUDE_DIR, CLOUDIFY_USER)
 INITIAL_INSTALL_DIR = join(CLOUDIFY_HOME_DIR, '.installed')
+INSTALLED_COPONENTS = join(INITIAL_INSTALL_DIR, 'components.yaml')
+INSTALLED_PACKAGES = join(INITIAL_INSTALL_DIR, 'packages.yaml')
 INITIAL_CONFIGURE_DIR = join(CLOUDIFY_HOME_DIR, '.configured')
+CONFIGURED_COPONENTS = join(INITIAL_CONFIGURE_DIR, 'components.yaml')
```

Comment:
Is this a good coponent or a bad coponent? (same fix as @tehasdf 's from above basically)